config/initializers/omniauth.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+Rails.application.config.middleware.use OmniAuth::Builder do
+  if Rails.env.production?
+    client_options = {
+      identifier: ENV.fetch('SSO_PROD_ID'),
+      secret: ENV.fetch('SSO_PROD_SECRET'),
+      redirect_uri: "https://lea5.rezoleo.fr/#{AUTH_CALLBACK_PATH}"
+    }
+  elsif Rails.env.development?
+    client_options = {
+      identifier: ENV.fetch('SSO_DEV_ID'),
```

Comment:
With this way of fetching the env var, we need to set them up also when using the CLI (except for the tests). Meaning that they are required to generate controllers for instance, even if we don't need them.
If you have an alternative, I would gladly accept it